ui/app/src/modules/Search/Search.tsx

```diff
@@ -63,6 +63,7 @@ import { batchActions, dispatchDOMEvent } from '../../state/actions/misc';
 import { getStoredPreviewChoice } from '../../utils/state';
 import { getPreviewURLFromPath } from '../../utils/path';
 import ApiResponseErrorState from '../../components/ApiResponseErrorState';
+import ToolBar from './ToolBar';
```

Comment:
All new components add them to `src/components/{ComponentName}/{ComponentName,index}.{tsx,ts}` and name them uniquely.

For this particular instance for example:
- `src/components/SiteSearchToolbar/`
  - `SiteSearchToolbar.tsx`
  - `index.ts`